<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eventPage</class>
 <widget class="QWidget" name="eventPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ввод обращений</string>
  </property>
  <property name="toolTip">
   <string>Ввод обращений</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblAverageDuration">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Средняя продолжительность заболевания</string>
     </property>
     <property name="buddy">
      <cstring>edtAverageDuration</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="edtAverageDuration">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbDiagnosisTypeAfterFinalForm043">
     <item>
      <property name="text">
       <string>Сопутствующий</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Основной</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblFillDiagnosticsEventsResults">
     <property name="text">
      <string>Заполнять результаты осмотра и События в форме ввода</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblDiagnosisTypeAfterFinalForm043">
     <property name="text">
      <string>Предпочитаемый тип диагноза в форме ввода 043 после закл</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chkCheckDispanserObservation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="chkEventTypeActive">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="chkSurveillancePlanningDialog">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblEventTypeActive">
     <property name="text">
      <string>Учитывать актуальность типа События</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="chkFillDiagnosticsEventsResults">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblSurveillancePlanningDialog">
     <property name="text">
      <string>Включить планирование профилактического наблюдения</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="chkPlanningParam">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Автоматическое заполнение периодов планирования в ККДН</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblCheckDispanserObservation">
     <property name="text">
      <string>Контроль заполнения ДН в событии</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QGroupBox" name="grbParam">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblPlanningFreq">
        <property name="text">
         <string>Частота периода планирования (мес)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="edtPlanningFreq">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>36</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblPlanningBegDate">
        <property name="text">
         <string>Дата начала периода планирования</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbPlanningBegDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Первый день месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>От текущей даты</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblPlanningDuration">
        <property name="text">
         <string>Длительность периода планирования (дней)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="edtPlanningDuration">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>28</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
